.github/workflows/audit.yml

```
@@ -0,0 +1,22 @@
+name: Audit
+
+on:
+  pull_request:
+    paths:
+      - package.json
```

We should probably add yarn.lock as well

---

I know, but this is about looking for a signal of when the dependencies change, so we know to run the audit. And in development, we always update the lock file in a PR if we bump or add a dependency. You can even update the lock file without updating `package.json` - something we also want to catch, so the correct line here would be:

```suggestion
      - yarn.lock
```

---

I just realized that the suggested change is exactly that.